tests/api_tests/features/api_tests.feature

```diff
@@ -0,0 +1,95 @@
+@api
+Feature: Newsfeed API tests
+
+  Scenario: Check microservice status
+    When get request to /status/ is received
+    Then response status should be 200
+    And response should contain "OK" status
+
+  Scenario: Verify documentation
+    When get request to /docs/ is received
+    Then response status should be 200
+    And response should contain documentation
+
+  Scenario: Get events
```

Comment:
If didn't miss anything, this scenario will just return empty list. It might have sense to add a precondition for adding some events to feed `123` (like it's done for "delete" scenario).